app/javascript/components/student/editor/ExercismMonacoEditor.tsx

```diff
@@ -54,7 +55,9 @@ export function ExercismMonacoEditor({
     })
 
     MonacoServices.install(editor)
-    const url = normalizeUrl(`ws://localhost:3000/${language}`)
+    const url = normalizeUrl(
+      `ws://${process.env.LANGUAGE_SERVER_HOST}/${language}/${uuidv4()}`
```

Comment:
Probably want to also pass in the _protocol. I recommend the env has both `protocol` + `hostname` + `port` (so the entire origin).

Why?
- `wss`: secure web socket
- `ws`: regular (insecure) web socket

Reply:
Sounds good! I'll create an issue to migrate our servers to wss.